### PR TITLE
curd operations using jdbc

### PR DESCRIPTION
**Pull request by 100zenithQuokka500**

### CODE REVIEW

**Comment by H1mang1** [comment]
@rkpatra201

File: src/main/java/org/example/ConnectionJDBC.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
+package org.example;
+import java.sql.*;
```
Comment (rkpatra201) [inline src/main/java/org/example/ConnectionJDBC.java:2]: avoid wild card imports: https://stackoverflow.com/questions/3348816/intellij-never-use-wildcard-imports

File: src/main/java/org/example/ConnectionJDBC.java [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+public class ConnectionJDBC {
+    public static void main(String[] args) throws ClassNotFoundException{
+        String url="jdbc:mysql://127.0.0.1:3306/core_java";
+        String username="root";
+        String password="himangi";
+      //  String query="INSERT INTO employees(id,name,job_title,salary) VALUES (3,'Harshit','Full stack developer',87000.0)";
+       // String query="DELETE FROM employees";
+        String query="UPDATE employees SET job_title='Full stack Developer',salary=70000.0 WHERE id=2;";
```
Comment (rkpatra201) [inline src/main/java/org/example/ConnectionJDBC.java:10]: do code formatting.

File: src/main/java/org/example/ConnectionJDBC.java [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        try{
```
Comment (rkpatra201) [inline src/main/java/org/example/ConnectionJDBC.java:11]: use interface based design.

```java 

public interface EmployeeDao{

  public int save(Employee employee);
  public int update(Employee employee, int id);
  public int delete(int id);
  public Employee find(int id);
  public List<Employee> findAll();

}

```

Comment (rkpatra201) [reply src/main/java/org/example/ConnectionJDBC.java:11]: write implementation class for above using Java Jdbc

**Review by rkpatra201 — CHANGES_REQUESTED** [review]
Please work on suggested comments

**Pull request merged**